docs/use-github-in-bcgov/github-transition-guide.md

```diff
@@ -128,6 +128,10 @@ If needed, Zenhub has [more support](https://help.zenhub.com/support/solutions/a
 
 ### IDIRs
 
+### P2 IDIR login 
+
+Vendors using both **P2 IDIRs** and external emails must log in with **<IDIR>@gov.bc.ca**. WS_LIC_M365_P2/OCIO_LIC_M365_P2 is the Microsoft license for Government contractors without an E5 IDIR, and need MFA and VPN **only**
```

Comment:
```suggestion
Vendors using both **P2 IDIRs** and external emails must log in with **\<IDIR\>@gov.bc.ca**. WS_LIC_M365_P2/OCIO_LIC_M365_P2 is the Microsoft license for Government contractors without an E5 IDIR, and need MFA and VPN **only**.
```

The IDIR is not showing in the docs as the `<>` tags are being interpreted by markdown. The brackets can be escaped by adding the `\` before them.

I don't have knowledge about the "WS_LIC_M365_P2/OCIO_LIC_M365_P2 is the Microsoft license for Government contractors without an E5 IDIR, and need MFA and VPN **only**" to know if that is correct or not. Maybe @jenreiher-bcgov knows if that is the correct license?

FYI: You can [preview PR changes on the dev site](https://dev.developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/#p2-idir-login)

Reply:
@MonicaG @Shivanganii This second part feels like "how the sausage is made" - I don't think any of our users reading this need to know that, it's more for troubleshooting stuff for 77000

Just the first sentence feels like enough, but maybe give an example of an external email would make it stronger? e.g. `ggg123@gmail.com`